cds-snc/platform-forms-client
# Expanding Form Name Input

# Summary | Résumé

Partial fix for #2003
![expanding-input](https://user-images.githubusercontent.com/62242/236016325-8b2d01b9-f8e2-4b0c-9eb4-b70b6e02a86a.gif)


###  Form Title
Adds Expanding input component for Form Title

Demo input with long title 👉  https://www.youtube.com/watch?v=RpJugHjWxlk

###  My Forms
Updates Card grid to handle really long titles 
- Adjust line clamp to 2 lines 
- [Remove inline-block ... this seems to have been blocking the ... from appearing](https://github.com/cds-snc/platform-forms-client/pull/2025/files#diff-91bbc2179640ca0d6ecae8b2a9d0b81c98d3972cde241f713e778120000b2f48L130)
<img width="528" alt="Screenshot 2023-05-03 at 2 19 31 PM" src="https://user-images.githubusercontent.com/62242/236009020-77b69483-076d-42b8-a6ec-9526a5b3c27b.png">

### Form Title 
<img width="500" alt="Screenshot 2023-05-03 at 2 38 03 PM" src="https://user-images.githubusercontent.com/62242/236012563-31ea1b18-b0ce-40c5-9cae-cbeaabc25905.png">

### Notes
See before screens https://github.com/cds-snc/platform-forms-client/pull/2025#issuecomment-1533526624

# Test instructions 
- Create a new form
- Add a really long form name --- to stretch the input to multiple lines
- Check My Form form title output + Form title output 


# Pull Request Checklist

Please complete the following items in the checklist before you request a review:
